scripts/sharedFunctions.js

```diff
@@ -208,12 +208,18 @@ async function updateLocalStorage(manga, options) {
 			manga.lastMangaDexChapter = manga.currentChapter.chapter;
 		}
 	}
-	await storageSet(manga.mangaDexId, {
+	nManga = {
```

Comment:
Any ``undefined`` field is not saved, you should make fields ``undefined`` and let the browser not save them instead of adding conditions, but it has the same result.